shared/constants.js

```diff
@@ -30,45 +30,72 @@ var VERSION = '6.0.0';
 module.exports = {
     version: VERSION,
 
-    tools: {
-        gitMinVersion: '2.13',
-        npmMinVersion: '3.10',
-        podMinVersion: '1.2'
-    },
-    
-    cordova: {
-        pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#dev',    // dev
-        //pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#v' + VERSION, // GA
-        minimumCliVersion: '7.0.0',
-        platformVersion: {
-            ios: '4.4.0',
-            android: '6.2.3'
-        }
-    },
+    templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#dev',    // dev
+    //templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#v' + VERSION, // GA
 
-    appTypes: {
-        ios: ['native', 'native_swift', 'react_native', 'hybrid_local', 'hybrid_remote'],
-        android: ['native', 'native_kotlin', 'react_native', 'hybrid_local', 'hybrid_remote']
-    },
-
-    templates: {
-        repoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#dev',    // dev
-        //repoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#v' + VERSION, // GA
-        appTypesToPath: {
-            ios: {
+    forceclis: {
```

Comment:
Less code / more config (coming out of constants.js) e.g. the tools that a given force cli depend on, the platforms/apptypes it supports, the IDE it's generating an app for etc. All of those are defined in constants.js.